mozilla-release/browser/base/content/tabbrowser.xml

```diff
@@ -2045,6 +2045,11 @@
             var evt = new CustomEvent("TabOpen", { bubbles: true, detail });
             t.dispatchEvent(evt);
 
+            // Go private before navigating if requested.
+            if (aPrivate) {
+              t.private = true;
+            }
+
```

Comment:
Same here. Please, add notes and comments when you change the location for a code block. For merging process we must know what is was where and why. In comment please add CLIQZ and ticket number (reason for moving a block will be useful too)